.github/workflows/release.yml

```diff
@@ -0,0 +1,25 @@
+
+on:
+  push:
+    tags: ["testing-[0-9].[0-9].[0-9]"]
```

Comment:
Does the `"testing-` mean that this will only match when the push contains a commit with a tag named "testing-..."?

Reply:
Yeah, I apparently left this one in.